pkg/frontend/openshiftcluster_preflightvalidation.go

```diff
@@ -88,26 +102,36 @@ func (f *frontend) _preflightValidation(ctx context.Context, log *logrus.Entry,
 		}
 	}
 
-	converter.ToInternal(ext, oc)
-	if err := staticValidator.Static(ext, nil, f.env.Location(), f.env.Domain(), f.env.FeatureIsSet(env.FeatureRequireD2sV3Workers), resourceID); err != nil {
-		return api.ValidationResult{
-			Status: api.ValidationStatusFailed,
-			Error: &api.ManagementErrorWithDetails{
-				Message: to.StringPtr(err.Error()),
-			},
+	switch {
+	case isCreate:
+		if err = staticValidator.Static(ext, nil, f.env.Location(), f.env.Domain(), f.env.FeatureIsSet(env.FeatureRequireD2sV3Workers), resourceID); err != nil {
```

Comment:
In our RP openshiftcluster_putorpatch, we do quota validation as well on create.  would/should we consider this?

Reply:
This was looked into for the previous PR and unfortunately this isn't something that can be done during preflight.